### PR TITLE
Added ButterflyFaithfulMonomorphismIntoSimplicialSurface and AllButterflyFaithfulMonomorphismsIntoSimplicialSurface

### PR DESCRIPTION
**Pull request by FlynnFehre**

### CODE REVIEW

File: gap/Morphisms/morphisms.gi [hunk 1]
```diff
@@ -478,8 +478,523 @@ InstallMethod( InverseGeneralMapping, "for a polygonal morphism",
...
+                        tmp[list1[i]] := bijection[i];
+                    od;
+                    Add(res, tmp);
+                od;
+                # res := List(AllPermutations, x -> IndexInducedMap(list1, x));
+                return res;
+            fi;
+        end;
+
+        MyImage := function(map, x)
```
Comment (MeikeWeiss) [inline gap/Morphisms/morphisms.gi:515]: Do you use this method somewhere?

**Review by MeikeWeiss — CHANGES_REQUESTED** [review]
I did not read everything yet, but it would be nice if you can delete all code that is commented out.

File: gap/Morphisms/morphisms.gi (hunk 1 above)
Comment (FlynnFehre) [reply gap/Morphisms/morphisms.gi:515]: Oh, yeah I will delete the unnecessary comments. Also, no I do not use the function anywhere. I will remove it.

Comment (FlynnFehre) [reply gap/Morphisms/morphisms.gi:515]: Actually, there are a few more functions I do not use. I will remove them as well

Comment (FlynnFehre) [reply gap/Morphisms/morphisms.gi]: I have removed everything, compiled the manual and tested everything. Locally, everything worked fine. But the tests here are failing. The errors also seem to be weird. Is there a way to restart the tests here without a "no change"-push?

Comment (MeikeWeiss) [reply gap/Morphisms/morphisms.gi]: Yes, I restarted the test but nothing has changed. The error is really weird and also occurs at PR218 since yesterday.

Comment (MeikeWeiss) [reply gap/Morphisms/morphisms.gi]: It's a problem from GAP, it will be resolved soon and then we can restart the checks

**Comment by MeikeWeiss** [comment]
There is still this unnecessary file: doc/tikz-files/Image_convex_facegraph_Double6Gon.tex

**Review by MeikeWeiss — APPROVED** [review]
Nice!

**Pull request merged**